spec-0004/index.md

```diff
@@ -112,7 +112,7 @@ amount of due dilligence before giving people access. This is because once a pro
 access their work will be broadcasted through the broad exposure of Scientific Python. This
 could be abused to publish malicious packages.
 
-A project's chosen representative should each create an
+A project's chosen representatives should each create an
 account on https://anaconda.org and share their usernames with the
 project.
```

Comment:
I think this is the intention?
```suggestion
site admins.
```

Reply:
I actually read it differently: i.e. it is recommended that the representatives share the account names with the projects so that other members of the project are aware of the account(s) for uploading wheels.

Obviously it's still not very clear :upside_down_face:

Reply:
I would prefer that the manner and methods in which projects internally share information remain outside the scope of this SPEC. If indeed that is the authors' intent, then I would propose removing the "share their username" clause.

Reply:
So this is what I wanted to say.
```suggestion
admins of the Scientific Python organization on Anaconda.
```

For sharing the info within your own project, this is your call on how you administrate things. Here we just need to know who to give access to.